guide/blueprints/catalog/index.md

```diff
@@ -375,153 +375,120 @@ the latest non-snapshot version will be loaded when an entity is instantiated.
 ~~~
 ~~~ bash
-Goodbye
+echo Hello, World!
 ~~~
 
+Now modify `myfile.sh` to contain a different message, change the version number in `catalog.bom` to
+`1.1.0`, and use the BR command to send the bundle to the server.
+
+If you now deploy a new instance of the server using the same YAML as above, you should be
+able to confirm that the new script has been run (either by looking at the stdout of the launch task, or looking at the script itself)
+
 At this point, it is also possible to deploy the original `Hello, World!` version by explicitly stating
 the version number in the YAML:
 
 ~~~ yaml
 location: localhost
 services:
-- type: script-server:1.0.0
-~~~
-
-And we can now confirm that the original text file has been used:
-
-~~~ bash
-cat /tmp/brooklyn-martin/apps/bek7efltx8/entities/EmptySoftwareProcess_r4emcpg05y/files/myfile.txt 
-~~~
-~~~ bash
-Hello, World!
+- type: my-server:1.0.0
 ~~~
 
 To demonstrate the scoping, you can create another bundle with the following `catalog.bom`. Note the
-bundle name and entity id have been changed, but it still references the text file
+bundle name and entity id have been changed, but it still references a script with the same name.
 
 ~~~ yaml
 brooklyn.catalog:
-  bundle: different-script-server
+  bundle: DifferentServerBundle
   version: 1.0.0
-  items:
-  - id: different-script-server
-    itemType: entity
-    item:
-      type: server
-      brooklyn.config:
-        files.runtime:
-          classpath://myfile.txt: files/myfile.txt
+  item:  
+    id: different-server
+    type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
+    brooklyn.config:
+      files.runtime:
+        classpath://myfile.sh: files/myfile.sh
+      launch.command: |
+        chmod +x ./files/myfile.sh
+        ./files/myfile.sh
+        
+      checkRunning.command:
+        echo "Running"  
 ~~~
 
-Now create a new `myfile.txt` file with a different message, and zip it along with the new `catalog.bom`.
+Now create a new `myfile.sh` script with a different message, and use the BR command to send it to Brooklyn.
 This can then be posted to the Brooklyn server as follows:
 
-~~~ bash
-curl -X POST -u admin:password -H "Content-Type: application/zip" --data-binary @/tmp/zip2/different-server.zip "http://127.0.0.1:8081/v1/catalog"
-~~~
-
-Now deploy a blueprint which deploys both the old and the new script servers
+Now deploy a blueprint which deploys both the old and the new script servers. Each of the three deployments will utilise the script that was included with there bundle.
 
 ~~~ yaml
 location: localhost
 services:
-- type: script-server:1.0.0
-- type: script-server:1.1.0
-- type: different-script-server
+- type: my-server:1.0.0
+- type: my-server:1.1.0
+- type: different-server
 ~~~
 
-**Note**: Both entities copy a file from `classpath://myfile.txt`, but as they are in different bundles, we should expect the files copied to the server to be different
-
-~~~ bash
```

Comment:
Just thought it was a bit too verbose- I would expect the reader to be able to check the files themselves for differences